apps/framework-cli/src/utilities/capture.rs

```diff
@@ -88,6 +90,13 @@ pub struct MooseActivity {
     pub ip: Option<String>,
 }
 
+fn serialize_uuid<S>(id: &Uuid, serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
a comment or a more descriptive function name why we need this specific function e.g. `serialize_uuid_as_str` `// serialized as_bytes by default for some serializers`